tests/v2/actions/workloads/deployment/verify.go

```diff
@@ -34,3 +37,52 @@ func VerifyDeployment(steveClient *steveV1.Client, deployment *steveV1.SteveAPIO
 
 	return err
 }
+
+// VerifyRolloutHistoryDeployment is a helper to validate rollout history deployment
+func VerifyRolloutHistoryDeployment(client *rancher.Client, clusterID, namespaceName string, deploymentName string, expectedRevision string) error {
+	wranglerContext, err := client.WranglerContext.DownStreamClusterWranglerContext(clusterID)
+	if err != nil {
+		return err
+	}
+
+	latestDeployment, err := wranglerContext.Apps.Deployment().Get(namespaceName, deploymentName, metav1.GetOptions{})
+	if err != nil {
+		return err
+	}
+
+	if latestDeployment.ObjectMeta.Annotations == nil {
+		return errors.New("revision empty")
+	}
+
+	revision := latestDeployment.ObjectMeta.Annotations[revisionAnnotation]
+
+	if revision == expectedRevision {
+		return nil
+	}
+
```

Comment:
can you switch this to the inverse case just so that the structure is in line with other verifies? i.e. 
```
if revision != expected {return new error}
return nil
```